gpt_computer_agent/standard_tools.py

```
@@ -312,4 +265,4 @@ def get_standard_tools():
 
 
 if __name__ == "__main__":
-    print(ask_to_user("What is your age"))
+    print(ask_to_user("What is your age"))
```

Ruff (0.8.2)

Undefined name `ask_to_user`

(F821)